tests/v2/actions/workloads/deployment/deployment.go

```diff
@@ -88,3 +97,41 @@ func UpdateDeployment(client *rancher.Client, clusterID, namespaceName string, d
 
 	return updatedDeployment, err
 }
+
+// RolbackDeployment is a helper to rollback deployments
+func RollbackDeployment(client *rancher.Client, clusterID, namespaceName string, deployment *appv1.Deployment, revision int) (string, error) {
+	deploymentCmd := fmt.Sprintf("deployment.apps/%s", deployment.Name)
+	revisionCmd := fmt.Sprintf("--to-revision=%s", strconv.Itoa(revision))
+	execCmd := []string{"kubectl", "rollout", "undo", "-n", namespaceName, deploymentCmd, revisionCmd}
+	logCmd, err := kubectl.Command(client, nil, clusterID, execCmd, "")
+	return logCmd, err
+}
+
+// GetRolloutHistoryDeployment is a helper to get rollout history deployment
+func GetRolloutHistoryDeployment(client *rancher.Client, clusterID, namespaceName string, deployment *appv1.Deployment) ([]string, error) {
```

Comment:
consolidating my other comments; maybe this could be validateDeploymentRolloutHistory(client, clusterID, namespace, deploymentName, expectedRolloutHistory) error {
...
}
since we usually don't have `get` function helpers as actions

Reply:
last point here but won't block on it; if you only ever `GetRolloutHistoryDeployment` then immediately `validateRolloutHistory` then  I would vote to just loop it all into one function since we usually don't add actions that only `Get`. does that make sense?